src/select.coffee

```diff
@@ -179,9 +179,15 @@ Ember.AddeparMixins.ResizeHandlerMixin,
       context: Ember.computed.alias 'controller.selection'
   .property 'itemView'
 
-  selectedLabel: Ember.computed ->
-    Ember.get @get('selection'), @get('optionLabelPath')
-  .property 'selection', 'optionLabelPath'
+  init: ->
+    @_super()
+    @selectionLabelPathChanges()
+
+  selectionLabelPathChanges: Ember.observer ->
```

Comment:
`selectionLabelPathChanges: Ember.on 'init', Ember.observer ->` and you can delete `init`